### DISCUSSION TITLE
Local video not working on android 14

### DISCUSSION THREAD

**Discussion by devnos** [flet-dev/flet#4392] [Q&A]
Greetings. Has anyone encountered the problem of playing video on a physical device running Android 14? There is a problem when playing local video, there is sound and no image. If anyone knows how to fix it, please write

**Comment by Muddassir-Farooq-official** [comment]
Set ft.video quality medium or low and then try

**Comment by devnos** [comment]
I tried setting filter_quality="low". Doesn't work

**Comment by Muddassir-Farooq-official** [comment]
Try Medium

**Comment by devnos** [comment]
I tried it, it doesn’t work, the bootloader in the form of a ring spins endlessly on the player

**Comment by ndonkoHenri** [comment]
Please when opening an issue make sure to use the right template.
Can you please share your Flet version, and steps to reproduce the issue?
A screen recording will also be useful.
